deltas/tokenizers/lexikon_chars.py

```diff
@@ -0,0 +1,47 @@
+# Unified Ideographs, Unified Ideographs Ext A,
```

Comment:
"lexicon_chars.py"

Reply:
changed name from lexikon_chars to lexicon_chars